src/app/login-dialog/login-dialog.component.scss

```diff
@@ -0,0 +1,55 @@
+.login {
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__heading {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 25px;
+    margin-bottom: 20px;
+
+    & p {
```

Comment:
don't use tags to define property. Here p tag is directly being used, create a class for the same, please make this change in all css files, there should't be any tags being directly used in any css files, like: p,img h1.. h7 etc

Reply:
Fixed that